homeassistant/components/androidtv/config_flow.py

```diff
@@ -313,6 +312,35 @@ def _async_apps_form(self, app_id):
             },
         )
 
+    async def async_step_commands(self, user_input=None):
+        """Handle options flow for custom comands."""
+        if user_input is None:
+            return self._async_commands_form(self._conf_cmd_id)
+
+        cmd_value = user_input.get(CONF_CMD_VALUE)
+        if not cmd_value:
+            self._cust_commands.pop(self._conf_cmd_id, None)
+        else:
+            self._cust_commands[self._conf_cmd_id] = cmd_value
+
+        return await self.async_step_init()
+
+    @callback
+    def _async_commands_form(self, cmd_id):
+        """Return configuration form for custom commands."""
+        data_schema = {
+            vol.Optional(
+                CONF_CMD_VALUE,
+                description={"suggested_value": self._cust_commands.get(cmd_id, "")},
```

Comment:
Should `suggested_value` be `current_value`?

Reply:
`suggested_value` propose the current configured value in the cell but allow the user to leave it empty. I don't know the keyword `current_value`.

Reply:
OK. I thought you came up with `suggested_value`, I didn't realize that you were adhering to an existing standard.